src/applications/disability-benefits/all-claims/pages/unemployabilityAdditionalInformation.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const additionalInformationDescription = (
+  <div>
+    <p>
+      If there is any other information you would like to provide as part of
+      your claim, please add it here.
+    </p>
+    <p>
+      It‘s important to tell us why your service-connected disability is the
+      reason you‘re no longer able to work. Providing specific examples will
+      help us understand your claim.
+    </p>
+    <p>
+      If you‘ve left one or more jobs because of your service-connected
+      disabilities, please note that.
+    </p>
+  </div>
+);
+
+export const uiSchema = {
+  'ui:title': 'Individual Unemployability',
+  'ui:description': additionalInformationDescription,
+  remarks: {
```

Comment:
Hmm this seems _very_ similar to what we have for 781/781a, maybe we can use a different key or nest it underneath an `unemployability` object?

Reply:
I thought so too, but that's the key name in the schema.

Reply:
gotcha, Can we nest it underneath the `unemployability` object like this:
```
export const schema = {
  type: 'object',
  properties: {
    unemployability: {
      type: 'object',
      properties: {
        hospitalizationHistory,
      },
    },
  },
};
```

Reply:
changed!